test/performance/odp_timer_accuracy.c

```diff
@@ -1008,6 +1010,37 @@ static void print_stat(test_global_t *test_global)
 	print_nsec_error("max", max, res_ns, -1, -1);
 
 	printf("\n");
+
+	if (test_global->common_options.is_export) {
+		if (test_common_write("num after,num before,num exact,num retry,"
+				      "error after min (nsec),error after min resolution,"
+				      "error after max (nsec),error after max resolution,"
+				      "error after ave (nsec),error after ave resolution,"
+				      "error before min (nsec),error before min resolution,"
+				      "error before max (nsec),error before max resolution,"
+				      "error before ave (nsec),error before ave resolution,"
+				      "final timeout error max (nsec),"
+				      "final timeout error max resolution\n"))
+			goto exit;
```

Comment:
Perhaps better not to use `goto` here and do everything export related inside this `if` branch.